Add custom message to response code assertions

jadedcore opened the pull request:
It would be nice to be able to pass in a custom message to the response code convenience methods.  Especially since it is already supported in the protected method _assertStatus.

jadedcore committed:
Add custom message to response code assertions

It would be nice to be able to pass in a custom message to the response code convenience methods.  Especially since it is already supported in the protected method _assertStatus.

markstory commented on src/TestSuite/IntegrationTestCase.php:
```diff
@@ -702,39 +702,51 @@ public function viewVariable($name)
      * @return void
      */
-    public function assertResponseOk()
+    public function assertResponseOk($message)
```
These new parameters have to be optional parameters for backwards compatibility. `$message = null` would be a good default.

jadedcore committed:
Added default null value to $message

Added the requested default null value to $message in order to maintain backwards compatibility.

jadedcore committed:
Added the @param comments

Added the @param comments to all of the changed methods.

markstory commented:
@jadedcore Looking good. Only a few lint errors left and this is good to go.

jadedcore committed:
Removed Whitespace

markstory commented:
Thanks!
